site/docs/exec-groups.md

```diff
@@ -162,3 +170,36 @@ my_rule = rule(
 #
 ```
 
+### Execution groups and platform execution properties
+
+It is possible to define `exec_properties` for arbitrary execution groups on
+platform targets (unlike `exec_properties` set directly on a target, where
+properties for unknown execution groups are rejected). Targets then inherit the
+platform's `exec_properties` that affect the default execution group and any
```

Comment:
"Targets then inherit the execution platform's `exec_proeprties`"

Reply:
Done (here and in one other place) - not sure if it should be made clearer.